Sources/CrossCompilationDestinationsTool/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ target_link_libraries(CrossCompilationDestinationsTool PUBLIC
 # NOTE(compnerd) workaround for CMake not setting up include flags yet
 set_target_properties(CrossCompilationDestinationsTool PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
+  
+
```

Comment:
I suppose that we should file an issue and tag it here so that we can clean this up.

Reply:
Done in https://github.com/apple/swift/issues/64900

Reply:
can we make this conditional on windows + add a comment why we need this?